lib/console/teams/membership_resolver.ex

```diff
@@ -6,10 +6,12 @@ defmodule Console.Teams.MembershipResolver do
       Ecto.assoc(current_team, [:memberships])
       |> Repo.paginate(page: page, page_size: page_size)
 
-    entries = memberships.entries |> Repo.preload(:user)
+    entries = memberships.entries |> Repo.preload([user: [:twofactor]])
```

Comment:
this works. if you want to be fancy, you could do something like how notifications gets its virtual `active` column. might be a little more efficient. your call though

Reply:
it would be 1 query instead of 3 (I think) and you wouldn't need to load all the associated fields for users and twofactors into memory